tests/testthat/test-commit.R

```diff
@@ -72,3 +72,35 @@ test_that("creating a commit in another directory without author works", {
   sig <- git_signature_default(repo)
   expect_equal(log$author, git_signature_info(sig)$author)
 })
+
+test_that("status reports a conflicted file", {
+  # temporary measure until gert can do a non fast forward merge
+  skip_if_not_installed("git2r")
```

Comment:
I know this is a non-starter but I have to be able to *create* a merge conflict and at the moment I can't with gert alone, which is #41.

Reply:
Totally fine as a for now. I don't mind at all adding `git2r` to Suggests, it might be useful for some other cross-validation tests between gert/git2r as well.

Reply:
OK added git2r to Suggests, at least for a little while.